.github/workflows/pr-to-master.yaml

```diff
@@ -0,0 +1,22 @@
+name: pr-to-master
```

Comment:
Just a minor comment, maybe we could rename the file and name here to "verify-branch.yaml" and "verify-branch".